zer0-os/zOS
# feat(redesign-conversation-header): update header components and styles

### What does this do?
- updates header components and styles

### Why are we making this change?
- as part of redesign

### How do I test this?
- run tests as usual
- run UI > check conversation header 

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?


Before
<img width="1440" alt="Screenshot 2024-09-11 at 10 48 42" src="https://github.com/user-attachments/assets/f971c089-73ef-44e0-a6df-eacdd2701c6c">





After
<img width="1440" alt="Screenshot 2024-09-11 at 10 48 03" src="https://github.com/user-attachments/assets/a07b0259-4f84-4895-a82b-68caadf174a6">
